source/ed/ed_reader.cpp

```diff
@@ -1072,13 +1072,12 @@ void EdReader::fill_vector_to_ignore(navitia::type::Data& , pqxx::work& work,
         if (nb_elt_in_component / principal_component->second >= min_non_connected_graph_ratio)
             continue; //big enough, we skip
 
-        uint64_t source = osmid_idex[vertex_idx];
+        uint64_t source = map_idx_to_id[vertex_idx];
```

Comment:
why don't you use the edge_to_ignore anymore ? 
If it is not useful anymore, remove it